src/components/Header.js

```diff
@@ -22,15 +22,17 @@ class Header extends React.Component {
     super(props);
     const { description, image, twitterHandle, title } = props;
     const meta = {
-      'twitter:site': 'opencollect',
-      'twitter:creator': twitterHandle,
+      'twitter:site': '@opencollect',
+      'twitter:creator': twitterHandle ? `@${twitterHandle}` : ' ',
```

Comment:
why a non empty string?

Reply:
At the root dir, it returns undefined.

Reply:
yes an empty string but why a space? Return `''`, not `' '`;

Reply:
will fix